cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -0,0 +1,215 @@
+1. Click **Save**.
+
+## Configure advanced settings
+
+The following sections describe the advanced settings you can configure for an SSO authentication method. To configure an authentication method:
+
+1. Log in to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) as an admin user.
+1. Go to **Organization** > **Authentication**.
+1. To configure an authentication method, click its name.
+1. At the top of the page, click **Edit**.
+1. Click **Advanced Settings**.
+1. Configure the advanced settings as desired.
+1. Click **Save**.
+
+### Allowed email domains
+
+By default, users can access your {{ site.data.products.db }} organization from any email domain. To restrict access to a specific list of email domains:
+
+1. Log in to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) as an admin user.
+1. Go to **Organization** > **Authentication**.
+1. To configure an authentication method, click its name.
+1. At the top of the page, click **Edit**.
+1. Set **Allowed Email Domains** to a comma-separated list of email domains. domains in CIDR format. Each domain should begin with a `@`. To ensure that you are not locked out of {{ site.data.products.db }} while you are configuring Cloud SSO, be sure to allow access to the email domain that you use to sign in.
+1. Click **Save**.
+
+### Autoprovisioning
+
+By default, autoprovisioning is disabled. In order to log in to {{ site.data.products.db }} using SSO:
+- A user's email address must exist in the SSO IdP, **and**
+- An admin user must have already invited the user to the {{ site.data.products.db }} organization.
+
+Autoprovisioning allows members to access your organization without an invitation. Users are assigned the [Developer role](/docs/cockroachcloud/console-access-management.html#developer) by default. If you enable autoprovisioning, Cockroach Labs recommends that you also limit the [allowed email domains](#allowed-email-domains) for the authentication method.
+
+To enable auto-provisioning for an SSO authentication method:
+
+1. Log in to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) as an admin user.
+1. Go to **Organization** > **Authentication**.
+1. Click the name of an authentication method.
+1. Click **Advanced Settings**.
+1. Next to **Autoprovisioning**, click **Enable**.
+
+## Add a custom authentication method
+
+You can add a custom authentication method to connect to any IdP that supports [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) or [OpenID Connect (OIDC)](https://openid.net/connect/).
+
+### OIDC
+
+To configure a custom OIDC authentication method:
+
+1. Log in to your IdP and gather the following information, which you will use to configure {{ site.data.products.db }} SSO:<ul><li>Issuer URL</li><li>Client ID</li><li>Client secret</li><li>Callback URL</li></ul>
```

Comment:
Yikes, perhaps we should log a docs tooling request to have this work in Markdown (if possible).

Reply:
There are some contexts that I just can't get sub-bullets to work. cc/ @nickvigilante